<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRoute_StopoverWindow</class>
 <widget class="QDialog" name="NewRoute_StopoverWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новий маршрут : зупинка</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="newroute_newstopover_notes_lb">
     <property name="text">
      <string>Нотатки</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QTextEdit" name="newroute_newstopover_notes_textEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QDateTimeEdit" name="newroute_newstopover_pdepature_time_dateTimeEdit">
     <property name="minimumDate">
      <date>
       <year>1792</year>
       <month>9</month>
       <day>15</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>dd HH:mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="newroute_newstopover_parrival_lb">
     <property name="text">
      <string>Час прибуття</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="newroute_newstopover_coordinates_lb">
     <property name="text">
      <string>Координати</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="newroute_newstopover_pdepature_lb">
     <property name="text">
      <string>Час відправлення</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDateTimeEdit" name="newroute_newstopover_parrival_time_dateTimeEdit">
     <property name="displayFormat">
      <string>dd HH:mm</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="newroute_newstopover_submit_buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="newroute_newstopover_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
